docs/operating-scylla/nodetool-commands/cleanup.rst

```diff
@@ -2,7 +2,7 @@
 
 Nodetool cleanup
```

Comment:
We don't implicitly state that the cleanup is now performed only on the local node by default. I'm not sure if it's clear. Perhaps we should mention it. We can for example add "- Ceanup is performed only on the local node unless the global flag is provided" after "In addition, the following should be noted:".